pages.de/common/django-admin.md

```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> Djangos Hilfstool für administrative Aufgaben.
+> Weitere Informationen: <https://docs.djangoproject.com/en/4.1/ref/django-admin/>.
+
+- Ein neues Django Projekt erstellen:
+
+`django-admin startproject {{projekt_name}}`
```

Comment:
```suggestion
`django-admin startproject "{{projekt_name}}"`
```

Optional fix: project name can be quoted if it can contain special glob characters for POSIX-compatible shells.